src/frontend/screens/Game/GamePage/components/InstalledInfo.tsx

```diff
@@ -57,6 +59,61 @@ const InstalledInfo = ({ gameInfo }: Props) => {
       wineVersion.type === 'crossover' ? wineCrossoverBottle : winePrefix
   }
 
+  const info = (
```

Comment:
moved this outside of the return so I can return it with different wrappers depending on the design we need